src/pages/home/report/ReportActionsView.js

```diff
@@ -340,7 +338,8 @@ class ReportActionsView extends React.Component {
             .sortBy('sequenceNumber')
             .filter(action => action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU
                     || action.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT
-                    || action.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED)
+                    || action.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED
+                    || action.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED)
```

Comment:
May be clean this up a bit. 

`_.contains(_.values(CONST.REPORT.ACTIONS.TYPE), action.actionName)`